content/documentation/distributed-tracing/index.adoc

```diff
@@ -20,51 +20,40 @@ toc::[]
 
 Kiali shows tracing of your applications. We currently support Jaeger version 1.9 and newer (link:https://www.jaegertracing.io/docs/1.12/frontend-ui/#embedded-mode[Read docs, window="_blank"]).
 
-icon:bullhorn[size=2x]{nbsp} To embed Jaeger, you must have the same protocol in Kiali and Tracing service.
+icon:bullhorn[size=2x]{nbsp} To have the tracing integration, you must set the configuration in external services / tracing.
 
-== Kiali and Tracing Service have the same protocol
+== Tracing url set
 
-When Kiali and Tracing service have the same protocol, we'll have the embedded feature in the website.
+When Tracing url is set, Kiali show a link to the Distributed Tracing in the menu.
 
-In the Distributed Tracing link in the menu we can see a full view for search the traces of our services.
+image::menu_external_link.png[Distributed Tracing View]
 
-image::distributed_tracing_same_protocol.png[Distributed Tracing View]
+== Tracing in_cluster_url set
 
+In each service we'll have a new tab to display the traces for the service in a scatter plot. (If we have error traces this tab will display them)
 
-In each service we'll have a new tab to display the traces for the service. (If we have error traces this tab will display them)
+image::service_traces_tab.png[Service View]
 
-image::service_view.png[Service View]
 
-We can interact with this embedded feature like the Jaeger UI
+We can interact with the tracing tool with the external links, for example "View Trace in Tracing" goes to the tracing tool trace view.
```

Comment:
"Tracing tool" here means Jaeger, right? I think it would be better to name it explicitly, also to avoid many repetitions with "trace" / "tracing" words. Something like:

Interactions with Jaeger UI is possible thanks to the external links displayed on this page. For example, "View Trace in Tracing" will open the following view: